crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -45,13 +52,11 @@ def checks
             )
           }
 
-          compute_resources = Api::Crowbar.compute_resources_status
-          ret[:compute_resources_available] = {
+          compute = Api::Crowbar.compute_status
+          ret[:compute_status] = {
```

Comment:
Yes, true. I've just thought it would be better to gather similar errors in one set